dist/index.ts

```diff
@@ -7,8 +7,7 @@ const app: Express = express();
 const port = process.env.PORT || 8080;
 
 const corsOptions = {
-  origin: ['*'],
-  method: ['GET']
+  origin: ['*']
```

Comment:
why are we updating this?

Reply:
I think i need to rebase the change. I did not change this file

Reply:
Done